1-exercises/A-array-find/exercise.js

```diff
@@ -4,7 +4,9 @@
 */
 
 // write your code here
+function longNameThatStartsWithA(names){
 
```

Comment:
Did you need more help to finish this? please make sure you ask for help in Slack.